include/libcyphal/transport/can/msg_rx_session.hpp

```diff
@@ -40,7 +41,7 @@ namespace detail
 /// NOSONAR cpp:S4963 for below `class MessageRxSession` - we do directly handle resources here;
 /// namely: in destructor we have to unsubscribe, as well as let delegate to know this fact.
 ///
-class MessageRxSession final : public IMessageRxSession, private IRxSessionDelegate  // NOSONAR cpp:S4963
+class MessageRxSession final : private IRxSessionDelegate, public IMessageRxSession  // NOSONAR cpp:S4963
```

Comment:
Now, `IRxSessionDelegate` base class finally is the very first in the inheritance list - this becomes possible by switching to `cetl::pmr::InterfacePtr`.